libr/include/r_anal.h

```diff
@@ -889,6 +889,8 @@ typedef struct r_anal_op_t {
 	RAnalHint hint;
 	RAnalDataType datatype;
 	int vliw; // begin of opcode block.
+	int omode;
```

Comment:
RAnalOp is a struct that provides information about 1 opcode. the obits/omode should be tied to the whole analysis session, not a single instruction

Reply:
Got it. I will move obits/omode into the RAnal struct instead.